api-proxy.typ

```diff
@@ -1,3 +1,5 @@
+#import "@preview/in-dexter:0.0.5": *
```

Comment:
I don't think we need this import in every file since they are all included, not imported. See https://typst.app/docs/reference/scripting/#modules

Importing it in main should be enough.

Reply:
I was wrong, we do actually need to import it in all the files we use it in. It's fine for now though.